README.md

```diff
@@ -16,6 +16,6 @@ a _set_ of _challenges_
 - koalas, [week-02](./week-02/koalas) 
 - mudkip, [week-01](./week-01/mudkip) [week-02](./week-02/mudkip) [week-03](./week-03/mudkip) [week-04](./week-04/mudkip) 
 - mwc441, [week-01](./week-01/mwc441) [week-02](./week-02/mwc441) [week-04](./week-04/mwc441) 
-- nint8835, [week-01](./week-01/nint8835) [week-02](./week-02/nint8835) [week-03](./week-03/nint8835) 
```

Comment:
everything looks good, wanna quickly run the readme gen again?

Reply:
Fixed